src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -25,6 +30,7 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
         // TODO: add log message about successful fetched data from DB
+        logger.info("Successful fetched data from DB. Params: userId = {}", userId);
```

Comment:
The log message in 'getAllProductsFromShoppingCart' method should be more informative. It mentions fetching data from DB, but there's no actual database operation here. If this is a stub for future DB access, it should be noted, otherwise, it can be misleading.